java/test/apps/FileLocationPaneTest.java

```diff
@@ -1,33 +1,41 @@
 package apps;
 
+import jmri.InstanceManager;
+import jmri.implementation.FileLocationsPreferences;
+import jmri.swing.PreferencesPanelTestBase;
 import jmri.util.JUnitUtil;
 
 import org.junit.jupiter.api.*;
 import org.junit.Assert;
 import org.junit.Assume;
+import org.mockito.Mockito;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  *
  * @author Paul Bender Copyright (C) 2017
  */
-public class FileLocationPaneTest {
-
-    @Test
-    public void testCTor() {
-        Assume.assumeFalse(java.awt.GraphicsEnvironment.isHeadless());
-        
-        FileLocationPane t = new FileLocationPane();
-        Assert.assertNotNull("exists",t);
-    }
+public class FileLocationPaneTest extends PreferencesPanelTestBase {
 
     @BeforeEach
     public void setUp() {
         JUnitUtil.setUp();
+        InstanceManager.setDefault(FileLocationsPreferences.class, Mockito.mock(FileLocationsPreferences.class));
+        prefsPanel = new FileLocationPane();
     }
 
-    @AfterEach
-    public void tearDown() {
-        JUnitUtil.tearDown();
+    @Override
+    @Test
+    public void isPersistant() {
+        assertThat(prefsPanel.isPersistant()).isTrue();
+    }
+
+    @Override
+    @Test
+    public void getPreferencesTooltip() {
+        // should this actually return null?
```

Comment:
We have not stipulated that all preferences tabs in a tabbed pane require a tooltip, so, yes, those classes that return null for the preferences tooltip should return null (the method that cares is `javax.swing.JTabbedPane.insertTab(...)` and it accepts null for a tooltip).

Reply:
Thanks.  Will Fix